docs/resources/validator-faq.md

```diff
@@ -98,10 +98,11 @@ Each validator receives revenue in proportion to its total stake. However, befor
 
 ###  IRISnet Valdiators Risks
 
--   **Unavailability**: Validators are expected to keep signing votes for making new blocks. If a validator’s signature has not been included in the last 20,000 blocks, 
-the validator will get slashed by 0.5% and remove from current validatorset for 2 days.
--   **Double Sign**：If someone reports on IRIShub that a validator signed two different blocks at the same height, this validator will get slashed on IRIShub by 1% and remove from current validatorset for 5 day.
--   **Censorship**：If someone reports that a validator includes invalid transactions in block, this validator will get slashed on IRIShub by 2% and remove from current validatorset for 7 day.
+-   **Unavailability**: Validators are expected to keep signing votes for making new blocks. If a validator’s signature has not been included in more than half of the last 40,000 blocks and get jailed and removed from current validatorset for 1 day
+-   **Double Sign**：If someone reports on IRIShub that a validator signed two different blocks at the same height, this validator will get jailed and removed from current validatorset for 2 days. Their bonded tokens will get slashed by 1%.
```

Comment:
If the protocol detects that a validator voted multiple different opinions about the same block (same height/round), or voted for different blocks at the same height/round, this validator will get jailed ...